rsmas_logging.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+class rsmas_logger():
+
+    format = "%(levelname)s - %(message)s"
+    console_handler, file_handler = None, None
+    logfile_name = None
```

Comment:
Do you mean to make `logfile_name` a static variable? Same with the `handler`s and `logger`.

Reply:
Shouldn't be static. The intention is that each script gets a separate instantiation of the rsmas_logger object with its own custom logger and handler objects, which would mean we want non-static variables.

Reply:
Good to know. I wasn't aware that was how instance variables worked in python classes.